students/WesWang/lesson3/slicing_lab.py

```diff
@@ -0,0 +1,38 @@
+#=========================================================================================
+#Name: Slicing Lab
+#Author: Wesley Wang
+#Date: 4/18/2018
+#=========================================================================================
+
+# Exchange first and last items
+def exchange_first_last(seq):
+  result = [seq[-1]] + seq[1:-1] + [seq[0]]
```

Comment:
better to use a length-one slice:
seq[-1:]

that will give you back a sequence of the same type.

This code will give you a list, which, if you are slicing, say, a string, would not be what you want!